xml/admin_saltcluster.xml

```diff
@@ -283,17 +283,15 @@ ses-min2   /dev/vdc  hdd  8192M  450575 True
     device lists for inspection by the user.
    </para>
    <para>
-    To apply OSD specification to your cluster, run
+    To apply OSD specification to your cluster, run:
    </para>
 <screen>&prompt.cephuser;ceph orch apply osd -i <filename>drive_groups.yml</filename></screen>
-   <tip>
-    <para>
-     You can use the <option>--dry-run</option> option together with the
-     <command>ceph orch apply osd</command> command to see a preview of actions
-     that will actually happen when running the command without the
-     <option>--dry-run</option> option.
-    </para>
-   </tip>
+   <para>
+    To see a preview of actions and test your application, you can use the
+    <option>--dry-run</option> option together with the <command>ceph orch apply osd</command>
+    command. For example:
+   </para>
+<screen>&prompt.cephuser;ceph orch apply osd --dry-run <filename>drive_groups.yml</filename></screen>
```

Comment:
I think it would be helpful to have example output so the user knows what they will see if it is successful. Thoughts, @smithfarm ?

Reply:
```
# ceph orch apply osd -i /tmp/osd.yml --dry-run
...
+---------+------+------+----------+----+-----+
|SERVICE  |NAME  |HOST  |DATA      |DB  |WAL  |
+---------+------+------+----------+----+-----+
|osd      |test  |mgr0  |/dev/sda  |-   |-    |
|osd      |test  |mgr0  |/dev/sdb  |-   |-    |
+---------+------+------+----------+----+-----+
```